frame/tips/src/lib.rs

```diff
@@ -597,7 +614,7 @@ impl<T: Config> Pallet<T> {
 				finder,
 				deposit,
 				closes: old_tip.closes,
-				tips: old_tip.tips,
+				tips: BoundedVec::truncate_from(old_tip.tips),
```

Comment:
Why does this migration trunc? Should it not just fail?  
Then you can increase the limit and re-run

Reply:
yup, you are right